src/pudl/transform/ferc714.py

```diff
@@ -1203,14 +1203,17 @@ def spot_fix_forecast_years_xbrl(df):
         expected range.
         """
         # [2025-aug kmm]
-        # there was some mixup and C004245 put text in their forecast year fields;
-        # excluding those records for now
+        # there was some mixup and C004245 (Salt River Project) put text in their
+        # forecast year fields in the 2024 report. they submitted a new filing with
+        # correct data, but filter_for_freshest_data isn't able to detect a match
+        # between the two sets of entries to be able to override the bad data, and
+        # keeps both sets of entries. we drop the bad entries manually here.
         text_in_year_mask = (
             (df.respondent_id_ferc714_xbrl == "C004245")
             & (df.report_year == 2024)
-            & (df.net_demand_forecast_mwh.isna())
+            & ~(df.forecast_year.isna() | df.forecast_year.str.startswith("2"))
```

Comment:
I think i understand the gist of all of this but can you add an explanation for why the forecast_year startswith does the trick??

Reply:
oh i'm now understanding i didn't see the ~ and i thought you were removing everything w/ a forecast_year starting w/ 2